src/leveling/leaderboard.go.tmpl

```diff
@@ -15,10 +15,11 @@
 	{{ $rank := $skip }}
 	{{ $display := "" }}
 	{{ range $users }}
+	        {{- $usr := or ( and (ne .User.String "#") .User.String ) .UserID }}
```

Comment:
Please use a branch for this. (As is, it's mildly unreadable at a glance.)
Also, tabs instead of spaces for consistency with the rest of the file. Thanks.